### ISSUE TITLE
Flatpak package

### ISSUE THREAD

**Issue by kupiqu** [fetacore/Infinitex#6]
Flatpak packages are nice for users as they are more transparent than appimages in the sense that are integrated in package managers (easy to install/uninstall and regular check for updates), and typically flatpak packages are more system aware (look and feel, global menu, colortheme, etc.).

**Comment by fetacore** [comment]
I agree with you. I refer you to [this](https://github.com/electron-userland/electron-builder/issues/512#issuecomment-433688522). As soon as electron builder supports it I will compile it.
Thank you for the interest in my project :)
